packages/js-example-app/package.json

```diff
@@ -5,8 +5,8 @@
   "private": true,
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build-index": "yarn run -T ejs src/index.ejs baseURL=. -o src/assets/index.html",
-    "build-index:prod": "yarn run -T ejs src/index.ejs baseURL=js-example-app -o src/assets/index.html",
+    "build-index": "yarn run -T ejs src/index.ejs componentsPrefix=adac baseURL=. -o src/assets/index.html",
```

Comment:
Is this intentional to have adac as prefix here?

Reply:
No actually, good spot. I'll replace this with blr